reference/specs/bindings/blobstorage.md

```diff
@@ -20,4 +20,27 @@ spec:
 - `storageAccessKey` is the Blob Storage access key.
 - `container` is the name of the Blob Storage container to write to.
 
-> **Note:** In production never place passwords or secrets within Dapr components. For information on securly storing and retrieving secrets refer to [Setup Secret Store](../../../howto/setup-secret-store)
\ No newline at end of file
+> **Note:** In production never place passwords or secrets within Dapr components. For information on securly storing and retrieving secrets refer to [Setup Secret Store](../../../howto/setup-secret-store)
```

Comment:
fix typo: `securly` to `securely`

Reply:
This typo is in any ref doc - I will update them all....